MilitaryElite/MilitaryElite/LieutenantGeneral.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MilitaryElite
+{
+    public class LieutenantGeneral : Private, ILieutenantGeneral
+    {
+        public List<ISoldier> CommandingPrivates { get; }
+
+        public LieutenantGeneral
```

Comment:
Shouldn't we be able to add privates rather than passing then to the constructor?